__tests__/app.test.js

```diff
@@ -57,3 +57,34 @@ describe("GET /api", () => {
   });
   
 
+  describe("GET /api/reviews/:review_id", () => {
+    test("status:200, responds with the specific requested review as an object, containing the appropriate properties", () => {
+      return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then((response) => {
+        const objectKeys = Object.keys(response.body.review);
```

Comment:
Because the review at review id 1 is in your seed data, you can be sure of its values. You should check if the review returned by the response matches the exact review you would expect. In this case it's totally fine to hard-code that expected review.